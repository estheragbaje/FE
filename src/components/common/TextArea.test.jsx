import React from 'react';
import { shallow, mount } from 'enzyme';

import TextArea from './TextArea';

describe('<TextArea />', () => {
  const props = {
    onChange: jest.fn(),
  };
  it('renders the TextArea component correctly', () => {
    const wrapper = shallow(<TextArea />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should call the mock onChange function', () => {
    const wrapper = mount(<TextArea {...props} />);

    wrapper.find('textarea').simulate('change', { preventDefault() {} });
    expect(props.onChange).toBeCalled();
  });
});
