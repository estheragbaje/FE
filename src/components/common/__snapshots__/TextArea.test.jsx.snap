// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextArea /> renders the TextArea component correctly 1`] = `
<Fragment>
  <label
    className="pb-3"
  />
  <textarea />
</Fragment>
`;
